README.md

```diff
@@ -34,40 +37,41 @@ git clone https://github.com/Azure-Samples/functions-quickstart-javascript-azd.g
 ## Run on your local environment
 
 ### Prepare your local environment
+1) Navigate to the new _functions-quickstart-javascript-azd_ folder.
 1) Create a file named `local.settings.json` and add the following:
-```json
-{
-  "IsEncrypted": false,
-  "Values": {
-    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
-    "FUNCTIONS_WORKER_RUNTIME": "node"
-  }
-}
-```
+    ```json
+    {
+      "IsEncrypted": false,
+      "Values": {
+        "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+        "FUNCTIONS_WORKER_RUNTIME": "node"
+      }
+    }
+    ```
 
 ### Using Functions CLI
-1) Open this folder in a new terminal and run the following commands:
+1) Open the _functions-quickstart-javascript-azd_ folder in a new terminal and run the following commands:
```

Comment:
you'll need to change this again to ./functions-quickstart-javascript-azd/src/functions or reuse the context that the user CD'd into it up front.